lte/gateway/deploy/agw_install.sh

```diff
@@ -26,13 +27,14 @@ fi
 echo "Making sure $MAGMA_USER user is sudoers"
 if ! grep -q "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" /etc/sudoers; then
   apt install -y sudo
+  adduser $MAGMA_USER
```

Comment:
I would run it in non interactive mode ` adduser --disabled-password --gecos "" $MAGMA_USER `